Userland/Games/GameOfLife/Board.h

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2021, Andres Crucitti <dasc495@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Vector.h>
+#include <LibGfx/Point.h>
+#include <stdio.h>
+
+class Board {
+public:
+    Board(size_t columns, size_t rows);
+    ~Board();
+
+    size_t total_size() const { return m_columns * m_rows; }
+    size_t columns() const { return m_columns; }
+    size_t rows() const { return m_rows; }
+
+    size_t calculate_index(size_t column, size_t row) const { return row * m_columns + column; };
+
+    void toggle_cell(size_t index);
+    void toggle_cell(size_t column, size_t row);
+
+    void set_cell(size_t column, size_t row, bool on);
+    void set_cell(size_t index, bool on);
+
+    bool get_cell(size_t column, size_t row) const;
+    bool get_cell(size_t index) const;
```

Comment:
Please remove the superfluous `get_` prefix, per the [project coding style](https://github.com/SerenityOS/serenity/blob/master/Documentation/CodingStyle.md#right-3)